cloudify/_compat.py

```diff
@@ -43,8 +45,10 @@ def reraise(exception_type, value, traceback):
         raise value.with_traceback(traceback)
 
     text_type = str
+    exec_ = getattr(__builtins__, 'exec')
```

Comment:
gotta do getattr, because `exec_ = __builtins__.exec` or `exec_ = exec` would be a syntaxerror when this is imported under py2